src/rqt_graph/rosgraph2_impl.py

```diff
@@ -414,8 +414,9 @@ def _node_refresh(self, node, bad_node=False):
         return updated
 
     def _node_uri_refresh(self, node):
-        current_nodes = \
-            {namespace + name for name, namespace in self._node.get_node_names_and_namespaces()}
+        current_nodes = {
+            namespace + (' ' if namespace.endswith('/') else '/') + name
```

Comment:
I removed the extra space in the `if` case in 5cf4ad8d2411c993d720255087592a38e9828a75.